addons/packages/whereabouts/0.5.0/bundle/config/overlay/overlay-daemonset-install.yaml

```diff
@@ -0,0 +1,40 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#@overlay/match by=overlay.subset({"kind":"ServiceAccount"})
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: whereabouts
+  namespace: #@ data.values.namespace
+
+#@overlay/match by=overlay.subset({"kind":"ClusterRoleBinding"})
+---
+apiVersion: rbac.authorization.k8s.io/v1
+subjects:
+  #@overlay/match by="name"
```

Comment:
Just a small thing, maybe we use the match in form of `key:value` to make it more specific. But I think it is ok if this match also works.

like #@overlay/match by=overlay.subset({"name": "whereabouts"})

Reply:
Sounds good, will update it.